core/app/modules/import-export/directories/base.php

```diff
@@ -95,9 +120,22 @@ final public function run_import( array $settings ) {
 			$meta_data[ $sub_directory_name ] = $sub_directory->run_import( $settings[ $sub_directory_name ] );
 		}
 
+		$is_root = array_key_exists( 'site-settings', $settings );
```

Comment:
There is a Root class? If no, try to find a better way to indicate the root.

Reply:
Fixed